tests/CMakeLists.txt

```diff
@@ -24,21 +29,28 @@ endif()
 
 if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
     if (OE_SGX)
-        add_subdirectory(abortStatus)
+        # The following group does not work in simulation under WIN32
```

Comment:
Consider revising to: "Following tests currently fail in Windows simulation mode"

Reply:
Sure.